go.mod

```diff
@@ -48,6 +50,7 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/scylladb/go-reflectx v1.0.1
 	github.com/shopspring/decimal v1.3.1
+	github.com/smartcontractkit/chainlink-solana v0.0.0-20211217161610-55e26699acf9
```

Comment:
I think the CI is failing because of this line. It's the old version and doesn't match the version in `develop`?